storage/storage.go

```
@@ -71,10 +153,9 @@ var once sync.Once
 func GetStorageInstance() *Storage {
 	once.Do(func() {
 		if storage == nil {
-			InitializeDriver()
+			_ = InitializeDriver(context.Background())
```

Ignoring error here?

---

Yes since this does not require explicitly to return the error. It will be handled within the respective spanner module methods.